packages.json

```diff
@@ -17162,5 +17162,20 @@
     "description": "libvlc bindings for Nim",
     "license": "MIT",
     "web": "https://github.com/Yardanico/nim-libvlc"
+  },
+  {
+    "name": "nimcoon",
+    "url": "https://njoseph.me/gitweb/nimcoon",
```

Comment:
I might be wrong, but this doesn't look like a correct url for this package; it lists all of your projects.

Reply:
git clones its fine

Reply:
Adding .git to the end of the url takes to the correct page. I will make the correction.